audonnx/core/api.py

```diff
@@ -34,6 +40,9 @@ def load(
             In legacy mode path to model ONNX file
         labels_file: YAML file with labels
         transform_file: YAML file with transformation
+        device_or_providers: set device
```

Comment:
Do you think it will happen often that we use providers?
I was thinking, if we should simply name it `device`, but stay with the possibility to handover a list of providers as well.

Reply:
You are right, the most common use-case will be `cpu` and `cuda`. So let's simplify to `device`.

Reply:
done

Reply:
Cool, looking forward to use this!